v1/statements/statements.gunk

```diff
@@ -0,0 +1,172 @@
+//         Info: openapiv2.Info{
+//                 Title:       "Statements API",
+//                 Description: "Provides CRUD operations on the statements resource.",
+//                 Version:     "1.0.0",
+//         },
+//         BasePath: "/v1",
+//         Schemes: []openapiv2.SwaggerScheme{
+//                 openapiv2.HTTPS,
+//         },
+//         Consumes: []string{
+//                 "application/json",
+//         },
+//         Produces: []string{
+//                 "application/json",
+//         },
+//         Responses: map[string]openapiv2.Response{
+//                 "400": openapiv2.Response{
+//                         Description: "Returned when the request body is malformatted or does not match the expected request.",
+//                 },
+//                 "401": openapiv2.Response{
+//                         Description: "Returned when the request does not contains the user's credentials.",
+//                 },
+//                 "403": openapiv2.Response{
+//                         Description: "Returned when the user does not have permission to access the resource.",
+//                 },
+//                 "500": openapiv2.Response{
+//                         Description: "Returned when an unexpected error occured on the server side.",
+//                 },
+//         },
+//         SecurityDefinitions: openapiv2.SecurityDefinitions{
+//                 Security: map[string]openapiv2.SecurityScheme{
+//                         "OAuth2": openapiv2.SecurityScheme{
+//                                 Type:             openapiv2.TYPE_OAUTH2,
+//                                 Flow:             openapiv2.FLOW_ACCESS_CODE,
+//                                 Description:      "Access to statements resources is restricted to authenticated users. The authentication process follows the Authorization Code flow.",
+//                                 AuthorizationURL: "/v1/auth",
+//                                 TokenURL:         "/v1/token",
+//                                 Scopes: openapiv2.Scopes{
+//                                         Scope: map[string]string{
+//                                                 "read":  "Grants read access",
+//                                                 "write": "Grants write access",
+//                                         },
+//                                 },
+//                         },
+//                 },
+//         },
+// }
+package statements
+
+import(
+	"github.com/gunk/opt/http"
+	"github.com/gunk/opt/openapiv2"
+
+	types "github.com/openbank/openbank/v1/types"
+)
+
+// Status define the status of a statment.
+type Status int
+
+const(
+	_ Status = iota
+	// Received is the value for a received statment.
+	Received
+	// Pending is the value for a pending statement.
+	Pending
+)
+
+type Statement struct{
+	// ID is the unique identifier of a statement.
+	ID string `pb:"1" json:"id"`
+	// Status is the status of the statement.
```

Comment:
please keep an empty line between documented fields, to help readability.

Reply:
do you meant like :
```
// ID is the unique identifier of a statement.
ID string `pb:"1" json:"id"`

// Status is the status of the statement.`
Status Status `pb:"2" json:"status"
```

Reply:
well noted, thanks